packages/dotcom/.changeset/quick-kiwis-attack.md

```diff
@@ -0,0 +1,5 @@
+---
+'@guardian/support-dotcom-components': major
```

Comment:
DCR is [already prepared for this change](https://github.com/guardian/dotcom-rendering/pull/12425), but technically it's a non-backwards compatible change, hence major